src/System.IO.FileSystem/tests/Directory/GetFileSystemEntries_str_str.cs

```diff
@@ -700,6 +710,29 @@ public void WindowsSearchPatternInvalid()
             });
         }
 
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public void WindowsSearchPatternInvalid_Core()
+        {
+            GetEntries(TestDirectory, "\0");
```

Comment:
this is expecting to succeed? if so should it verify it gets zero results for all these?

Reply:
Should succeed. I didn't really care what number of results- I'll give it some thought.